charts/collabora-online/0.0.1/test_values.yaml

```
@@ -0,0 +1,51 @@
+image:
+  repository: collabora/code
+  tag: 6.4.6.2
+  pullPolicy: IfNotPresent
+
+strategy:
+  type: Recreate
+
+service:
+  type: NodePort
+  port:
+    port: 9980
+    nodePort: 30980
+
+env:   
+  domain: nextcloud\.domain\.tld       
+  dictionaries: de_DE en_GB en_US es_ES fr_FR it nl pt_BR pt_PT ru              
+  username: admin
+  password: changeme
+  extra_params:
+  # cert_domain:
+  # server_name:
+  # DONT_GEN_SSL_CERT:
+
+#securityContext:
+#  privileged: true
+
+# persistence:
+#   config:
+#     enabled: false
+#     emptyDir: false
+#     ## Persistent Volume Storage Class
+#     ## If defined, storageClassName: <storageClass>
+#     ## If set to "-", storageClassName: "", which disables dynamic provisioning
+#     ## If undefined (the default) or set to null, no storageClassName spec is
+#     ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+#     ##   GKE, AWS & OpenStack)
+#     # storageClass: "-"
+#     # accessMode: ReadWriteOnce
+#     # size: 1Gi
+#     ## Do not delete the pvc upon helm uninstall
+#     # skipuninstall: false
+#     # existingClaim: ""
+
+# ## TrueCharts Config
+
+# appVolumeMounts:
+#   config:
+#     enabled: true
+#     emptyDir: true
+#     mountPath: "/etc/loolwsd"
```

This needs not to be commented, also end-of-file whiteline missing.
Yes, this might fail on the current build. But the underlaying issue needs to be resolved, not storage commented out.

```suggestion
appVolumeMounts:
  config:
    enabled: true
    emptyDir: true
    mountPath: "/etc/loolwsd"
```